test/standardizedToken.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const chai = require('chai');
+const { ethers } = require('hardhat');
+const {
+    Contract,
+    utils: { keccak256, toUtf8Bytes },
+} = ethers;
+const { expect } = chai;
+const { getRandomBytes32, expectRevert, getGasOptions } = require('./utils');
+const { deployContract } = require('../scripts/deploy');
+
+const StandardizedToken = require('../artifacts/contracts/token-implementations/StandardizedToken.sol/StandardizedToken.json');
+const StandardizedTokenProxy = require('../artifacts/contracts/proxies/StandardizedTokenProxy.sol/StandardizedTokenProxy.json');
+
+describe('StandardizedToken', () => {
+    let standardizedToken, standardizedTokenDeployer;
+
+    const name = 'tokenName';
+    const symbol = 'tokenSymbol';
+    const decimals = 18;
+    const mintAmount = 123;
+
+    let token;
+    let tokenProxy;
+
+    let owner;
+
+    before(async () => {
+        const wallets = await ethers.getSigners();
+        owner = wallets[0];
+
+        standardizedToken = await deployContract(owner, 'StandardizedToken');
+        standardizedTokenDeployer = await deployContract(owner, 'StandardizedTokenDeployer', [standardizedToken.address]);
+
+        const salt = getRandomBytes32();
+
+        const tokenAddress = await standardizedTokenDeployer.deployedAddress(salt);
+
+        token = new Contract(tokenAddress, StandardizedToken.abi, owner);
+        tokenProxy = new Contract(tokenAddress, StandardizedTokenProxy.abi, owner);
```

Comment:
you can create these via the factory, so you don't have to hardcode artifact paths